src/hooks/useOutsideClick/useOutsideClick.tsx

```diff
@@ -2,7 +2,7 @@ import React from 'react';
 
 export interface UseOutsideClickProps<T> {
     ref: React.RefObject<T>;
-    handler?: () => void;
+    handler?: (e: MouseEvent | TouchEvent) => void;
```

Comment:
let's make e optional `e?`. It looks like breaking change right now.

Reply:
No, it won't be a breaking change. It's like onClick in Button where you can skip this argument if there is no need to use it